package.json

```diff
@@ -2,6 +2,9 @@
   "name": "acorn-dynamic-import",
   "description": "Support dynamic imports in acorn",
   "main": "lib/index.js",
+  "files": [
```

Comment:
The “files” field is dangerous - the only way published files should ever be restricted is by npmignore.

Reply:
> This will break the critical use case of `npm install foo && npm explore foo && npm install-test”. Tests must be published with packages.

The `tests` directory is already being ignored by this project using `.npmignore` as can be seen here: https://github.com/kesne/acorn-dynamic-import/blob/master/.npmignore

Reply:
Then that use case is already broken, unfortunately, but the files field is still dangerous and should never be used.